ROADMAP.md

```diff
@@ -1,3 +1,63 @@
 # ROADMAP
 
 The [OpenEBS ROADMAP](https://github.com/openebs/openebs/blob/HEAD/ROADMAP.md) lives within the main repo.
+
+________________________________________
+
+## Roadmap change process
+
+For a CNCF sandbox project with a **small maintainer team**, the roadmap change process should be **lightweight, pragmatic, and efficient**, while still being transparent and community-driven. The process below balances inclusiveness with agility.
+________________________________________
+
+### 1. Principles
+
+- **Keep it Simple:** Avoid bureaucratic overhead while maintaining transparency.
+- **Consensus First:** Decisions should aim for agreement among maintainers, with voting as a fallback.
+- **Community Input, Maintainer Decision:** Contributors and users provide feedback, but maintainers make the final call.
+- **Code of Conduct Compliance:** Discussions follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+### 2. Change Request Process
+
+#### 2.1 Proposal Submission
+
+- **Where?** Submit a GitHub Issue or a Pull Request modifying docs/roadmap.md.
+- **What to include?**
+  - **Brief description** of the change (e.g., new feature, priority shift).
+  - **Why it matters** (business or technical impact).
+  - **Potential risks** (breaking changes, complexity, etc.).
+  - **Expected timeline** (if applicable).
+
+#### 2.2 Quick Community Discussion (Asynchronous & Lightweight)
+
+- Proposal is **open for discussion for at least 5 business days**.
+- Discussion happens via:
+  - **GitHub Issue comments** (preferred)
+  - Pull Request
+  - CNCF Slack
+  - Monthly community call (if timing aligns)
+- **Feedback is encouraged but optional** – if no objections are raised, the process moves forward.
+
+#### 2.3 Maintainer Decision
+
+- After discussion, maintainers assess the proposal.
+- **Approval Criteria:**
+  - If **66%+ of maintainers agree** and there are **no strong objections**, the change is accepted.
+  - If there's disagreement, maintainers discuss internally and **vote if needed** (66% majority wins).
+  - Decision is **documented on GitHub** (Issue/PR closed with final reasoning).
+
+#### 2.4 Implementation & Communication
+
+- **Once approved:**
+  - The roadmap ([OpenEBS ROADMAP](https://github.com/openebs/openebs/blob/HEAD/ROADMAP.md)) is updated.
+  - The change is shared in the next community update (GitHub Discussions, Slack, or meeting notes).
+
+### 3. Conflict Resolution
+
+- If a proposal is contested:
+  - Maintain a respectful discussion in line with the Code of Conduct.
+  - If needed, bring in an external CNCF mentor for mediation.
+  - If deadlocked, escalate to the CNCF TOC for a neutral decision.
+
+### 4. Continuous Process Review
+
+- Every **6 months**, maintainers **review and adjust** the roadmap change process based on project needs.
```

Comment:
Should this be
"Maintainers periodically review and ...." instead of Every 6 months?

Reply:
Is this suggestion discarded? It's fine, but just confirming as the conversation is marked as resolved, but the change hasn't gone in.